pkg/hooks/hooks_test.go

```diff
@@ -27,8 +27,13 @@ func TestExecutableHooksAreRun(t *testing.T) {
 
 	ioutil.WriteFile(path.Join(tempDir, "test1"), []byte("#!/bin/sh\necho $HOOKED_POD_ID > $(dirname $0)/output"), 0755)
 
+	// So PodFromPodHome doesn't bail out, write a minimal current_manifest.yaml
+	ioutil.WriteFile(path.Join(podDir, "current_manifest.yaml"), []byte("id: my_hook"), 0755)
```

Comment:
if this becomes too painful (I do see it's repeated three times) I might suggest putting it in a function